firmware/src/Conduit.cpp

```diff
@@ -12,6 +12,18 @@ device and decide what funciton to all is abstracted away entirely by this libra
 
 #include "Conduit.h"
 
+#define PREFIXED_NAME_CHARS   35
+#define STATIC_STRING_CHARS   64
+#define RECONNECT_DELAY_MS    5000
+#define CONNECT_DELAY_MS      500
+#define ROOT_NODE_NAME        "ROOT"
+#define DEVICE_PATH_SUFFIX    "/device"
```

Comment:
Lines 20-24 are values that should not necessarily be configurable because they will break the API contract with the conduit server. Please include these lines with a block comment that specifies these should not be changed.